distiller/apputils/image_classifier.py

```diff
@@ -268,6 +268,9 @@ def init_classifier_compression_arg_parser(include_ptq_lapq_args=False):
                         help='path to checkpoint to load weights from (excluding other fields) (experimental)')
     load_checkpoint_group.add_argument('--pretrained', dest='pretrained', action='store_true',
                         help='use pre-trained model')
+    load_checkpoint_group.add_argument('--from-pretrained', dest='pretrained_checkpoint_path', default='',
+                        type=str, metavar='PATH',
```

Comment:
What is the reason for renaming the argument? 
It tends to be more confusing if the command line argument and the variable in the code have different names.

Reply:
I just wanted to maintain the same style used for other input argument.
As I saw they renamed it for the `--resume-from` flag, I just renamed it the same way. Other input arguments consisting in two words are renamed in a similar way.

Reply:
Ok, if it is consistent, perfect.